Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'foundation-rails'
 gem 'simple_form'
 gem 'devise'
 gem 'haml'
+gem 'rails_12factor'
```

Comment:
Run `rake`. Then put this gem in a `group :production` block, and run `rake` again. Leave this gem in the production group.